docs/elements/sageone/authenticate.md

```diff
@@ -0,0 +1,275 @@
+
+You can authenticate with {{page.heading}} to create your own instance of the {{page.heading}} element through the UI or through APIs. Once authenticated, you can use the element instance to access the different functionality offered by the {{page.heading}} platform.
+
+{% include callout.html content="<strong>On this page</strong></br><a href=#authenticate-through-the-ui>Authenticate Through the UI</a></br><a href=#authenticate-through-api>Authenticate Through API</a></br><a href=#parameters>Parameters</a></br><a href=#example-response>Example Response</a>" type="info" %}
+
+## Authenticate Through the UI
+
+Use the UI to authenticate with {{page.heading}} and create an element instance. {{page.heading}} authentication follows the typical OAuth 2 framework and you will need to sign in to {{page.heading}} as part of the process.
+
+If you are configuring events, see the [Events section](events.html).
+
+To authenticate an element instance:
+
+1. Sign in to Cloud Elements, and then search for the element in our Elements Catalog.
+
+    | Latest UI | Earlier UI  |
+    | :------------- | :------------- |
+    |  ![Search](../img/Element-Search2.png)  |  ![Search](../img/Element-Search.png)  |
+
+3. Create an authenticated element instance.
+
+    | Latest UI | Earlier UI  |
+    | :------------- | :------------- |
+    | Hover over the element card, and then click __Create Instance__.</br> ![Create Instance](../img/Create-Instance.gif)  | Click __Add Instance__.</br> ![Search](../img/Add-Instance.png)  |
+
+5. Enter a name for the element instance.
+6. You can choose to authenticate with the default APIM Subscription Key, or you can enter your own. The APIM Subscription Key is the subscription Primary key that you noted in the [Service Provider Setup section](setup.html).
+7. Click __Create Instance__ (latest UI) or __Next__ (earlier UI).
+8. Optionally add tags in the earlier UI:
+     1. On the Tag It page, enter any tags that might help further define the instance.
+      * To add more than one tag, click __Add__ after each tag.
+      ![Add tag](../img/Add-Tag.png)
+     1. Click __Done__.
+8. Provide your Sage credentials, and then allow the connection.
+8. Take a look at the documentation for the element resources now available to you.
+
+## Authenticate Through API
+
+Authenticating through API is a multi-step process that involves:
+
+* [Getting a redirect URL](#getting-a-redirect-url). This URL sends users to the vendor to log in to their account.
+* [Authenticating users and receiving the authorization grant code](#authenticating-users-and-receiving-the-authorization-grant-code). After the user logs in, the vendor makes a callback to the specified url with an authorization grant code.
+* [Authenticating the element instance](#authenticating-the-element-instance). Using the authorization code from the vendor, authenticate with the vendor to create an element instance at Cloud Elements.
+
+### Getting a Redirect URL
```

Comment:
Just a side note, we should brainstorm how to communicate the authentication flow in a way that shows the series of steps required in order to create that element instance.